### DISCUSSION TITLE
Browser support

### DISCUSSION THREAD

**Discussion by elrumo** [retextjs/retext#88] [Q&A] [answered]
Is there any way in which retext could be used in the browser?

I found this commit made in 2021 [Remove browser build](https://github.com/retextjs/retext/commit/6300446aa7590cfdd4ede6e53e0e420110056eec), and I wonder if there is any way to use retext in the browser, and if not, do you know of any alternatives?

Thanks!

**Comment by wooorm** [comment, marked as answer]
Yes. See the readmes.
